packages/grafana-ui/src/components/Cascader/Cascader.tsx

```diff
@@ -262,11 +263,24 @@ export class Cascader extends PureComponent<CascaderProps, CascaderState> {
                 onKeyDown={this.onInputKeyDown}
                 onChange={() => {}}
                 suffix={
-                  focusCascade ? (
-                    <Icon name="angle-up" />
-                  ) : (
-                    <Icon name="angle-down" style={{ marginBottom: 0, marginLeft: '4px' }} />
-                  )
+                  <>
+                    {activeLabel !== '' && (
+                      <IconButton
+                        name="times"
+                        aria-label="Clear selection"
+                        onClick={(e) => {
+                          e.preventDefault();
+                          e.stopPropagation();
+                          this.setState({ rcValue: [], activeLabel: '', inputValue: '' });
+                          this.props.onSelect('');
+                        }}
+                      />
+                    )}
+                    <Icon
+                      name={focusCascade ? 'angle-up' : 'angle-down'}
+                      style={{ marginBottom: 0, marginLeft: '4px' }}
+                    />
+                  </>
```

Comment:
This only adds it to the selected value, but once you start typing it disappears because it uses the Select component then, should this option be also enabled when typing?

Reply:
> I see, yeah I don't think adding this button is worth the effort, it's more of a nice to have.

+1